channel_api.md

```diff
@@ -352,6 +352,11 @@ client fails to read data from the connection for two successive
 intervals, the connection will emit an error and close. It will also
 send heartbeats to the server (in the absence of other data).
 
+If you supply `0` as the `heartbeat` parameter (or defaults to `0`), the
```

Comment:
But really, you should not be disabling heartbeats. Like, seriously. Don't do it. Heartbeats [exist for a reason](http://www.rabbitmq.com/heartbeats.html).